templates/mlflow/pipeline.yaml

```diff
@@ -15,10 +15,10 @@ tasks:
     product: products/features/features.csv
 
   - source: scripts/fit.py
-    name: fit-
+    name: fit-[[model]]-[[n_estimators]]-[[criterion]][[learning_rate]]-[[params_names]]-[[track]]-[[mlflow]]-[[mlflow_tracking_uri]]
     # we need to turn this off because parameters depend on the type of model
     static_analysis: disable
-    product: products/report.ipynb
+    product: products/report-[[model]]-[[n_estimators]]-[[criterion]][[learning_rate]]-[[params_names]]-[[track]]-[[mlflow]]-[[mlflow_tracking_uri]].ipynb
```

Comment:
here we have a few parameters that are only configuration and we should not pass them as `[[placeholders]]`, please remove: params_names, track, and mlflow_tracking_uri.

also remove "mlflow" since that's not a parameter in the grid